api/client/src/main/java/org/projectnessie/client/auth/oauth2/AuthorizationCodeFlow.java

```diff
@@ -138,7 +161,7 @@ public Tokens fetchNewTokens() {
     } catch (ExecutionException e) {
       abort();
       Throwable cause = e.getCause();
-      LOGGER.error("Authentication failed: " + cause.getMessage());
+      LOGGER.error("Authentication failed: {}", cause.getMessage());
```

Comment:
nit: How about just `cause` for the log argument (`Exception.toString()`)... Since this is a generic handler we cannot be sure that the message alone is meaningful (e.g. it'll be `null` for `NullPointerException` in older JVMs).

Reply:
Ah, yes, I forgot about this IJ nuisance... nevermind.